components/31-molecules/video/_video.scss

```diff
@@ -21,3 +21,14 @@
     float: right;
   }
 }
+
+.audio-description {
+  .standalone-link:not(.no-icon) {
+    &::after {
+      top: 0;
+      bottom: 0;
+      margin-top: auto;
+      margin-bottom: auto;
+    }
+  }
+}
```

Comment:
In feite zou die link van zichzelf goed moeten staan. Hebben we dat probleem met de icon alignment nergens anders?

Reply:
Yes, klopt helemaal. Ik had het enkel voor dit icon gefixed omdat ik het nergens anders zag, maar ik merk nu net dat het probleem kwam omdat er een 'float' op de link stond, waardoor het icoon ook verkeerd gepositioneerd was. Die heb ik verwijderd nu en vervangen door een text-align: right op de audio-description, wat logischer is.